codebuild/bin/install_saw.sh

```diff
@@ -28,6 +28,11 @@ fi
 DOWNLOAD_DIR=$1
 INSTALL_DIR=$2
 
+if [ -x "$INSTALL_DIR/bin/saw" ]; then
+	echo "Already installed";
```

Comment:
Can we be more specific for logging, `Saw already installed at "$INSTALL_DIR/bin/saw`?

Reply:
added.